mapping-for-navigation/_posts/0000-01-09-modeling-interesections-for-map-navigation.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Modeling Intersections for Map Navigation
+---
+
+Capturing the various cases of Intersection modelling in OSM and the best practices to follow. The aim of this document is not go into the technicalities of traffic flow in a junction but rather to give a mapper clear cut instructions on how to model different intersections in OSM. Aiming to keep this document simple and easy to follow.
```

Comment:
Nit: use one form consistently:

🇺🇸 modeling
🇬🇧 modelling

Reply:
fixed to 🇺🇸 `modeling`